app/presenters/tree_builder_automate_simulation_results.rb

```diff
@@ -0,0 +1,86 @@
+class TreeBuilderAutomateSimulationResults < TreeBuilder
+  has_kids_for Hash, [:x_get_tree_hash_kids]
+  def initialize(name, type, sandbox, build = true, root = nil)
+    @root = root
+    super(name, type, sandbox, build)
+  end
+
+  private
+
+  def tree_init_options(_tree_name)
+    {
+      :full_ids => true,
+      :add_root => false,
+      :expand   => true,
+      :lazy     => false
+    }
+  end
+
+  def set_locals_for_render
+    locals = super
+    locals.merge!(:id_prefix     => "aesimulation_",
+                  :autoload      => false,
+                  :cfme_no_click => true,
+                  :onclick       => false)
+  end
+
+  def root_options
+    []
+  end
+
+  def x_get_tree_roots(count_only = false, _options)
+    objects = []
+    idx = 1
+    MiqXml.load(@root).root.each_element do |el|
```

Comment:
If `iqXml.load(@root).root.each_element` returns an Enumerator, you can do:
`[1,2,3].each.with_index do |e,i|; p [e,i]; end`